status-im/status-react
# [ISSUE #2391] Added support for ERC681

addresses #2391

### Summary:

Add support for ERC681 (improve payment request URI in QR code).

### Steps to test:
- Open Status
- Navigate wallet/request
- Make sure generated QR code are correct 
- Navigate wallet/send: make sure valid QR code are parsed (in particular QR code with non matching `chain-id` should be rejected)
- Step 3, etc.

Some QR code for `ropsten`:

__No value__
![no-value](https://user-images.githubusercontent.com/359723/32787606-3e5ea9b4-c957-11e7-9790-912b17095c8d.png)

__Value 5 ETHs__
![5eths](https://user-images.githubusercontent.com/359723/32787626-4a3fe540-c957-11e7-8e68-5d178d218600.png)


status: ready
